### PR TITLE
Add option to use a segmented OT

### PR DESCRIPTION
**Pull request by danielbattistini**

### CODE REVIEW

File: Detectors/Upgrades/ALICE3/TRK/simulation/include/TRKSimulation/TRKLayer.h [hunk 1]
```diff
@@ -27,6 +29,8 @@ class TRKLayer
   TRKLayer(int layerNumber, std::string layerName, float rInn, float zLength, float thick);
   ~TRKLayer() = default;
 
+  void setLayout(eLayout layout) { this->mLayout = layout; };
```
Comment (mconcas) [inline Detectors/Upgrades/ALICE3/TRK/simulation/include/TRKSimulation/TRKLayer.h]:
```suggestion
  void setLayout(eLayout layout) { mLayout = layout; };
```

**Review by mconcas — COMMENTED** [review]
Looks good to me in general, approving to start the tests.

**Review by mconcas — APPROVED** [review]

**Comment by mconcas** [comment]
@danielbattistini is this finalised? Can I merge it?

**Comment by danielbattistini** [comment]
Yes you can merge! Thanks!

**Pull request merged**